python/pyspark/sql/connect/functions.py

```diff
@@ -89,6 +90,10 @@ def col(col: str) -> Column:
 column = col
 
 
+def colRegex(col: str) -> Column:
```

Comment:
This one should be in dataframe.py. Refer to https://github.com/apache/spark/blob/af8dd411aa9d15bc59d09cf9959d4a57debc9635/python/pyspark/sql/dataframe.py#L1880

Reply:
I know it. But `def col(col: str) -> Column:` in functions.py.

Reply:
hmm I am not finding `def colRegex(col: str)` in functions.py in pyspark.sql. Did I miss something?

Reply:
Please see: https://github.com/apache/spark/blob/9b69331602eae5e8ddc969793c951e20b755582d/python/pyspark/sql/connect/functions.py#L85

Reply:
I know you meaning that follows `sql/dataframe.py`.

Reply:
yes!